### PR TITLE
[PyTorch] Fix AlexNet dropout ordering

### LINKED ISSUE #1131: Confusion regarding AlexNet dense layer ordering
@AnirudhDagar 

Problem: In the book, when AlexNet is implemented, in MxNet, it shows 

`nn.Dense(4096, activation="relu"), nn.Dropout(0.5)` which is understood to be in the order : 

dense layer -> relu -> dropout.

However, in pyTorch, the order seems to have changed and looks like this:

```
nn.Dropout(p=0.5),
nn.Linear(6400, 4096), nn.ReLU()
```

which seems like a different ordering than MxNet.

Can someone clarify this and edit the exact ordering accordingly. 

Thanks

### PR DESCRIPTION
**Pull request by AnirudhDagar**
*Description of changes:*
Closes #1131 

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by AnirudhDagar** [comment]
FYI @mli

**Comment by mli** [comment]
Job d2l-en/PR-1132/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1132/

**Comment by StevenJokes** [comment]
why uses `http` other than `https` in URL of preview version?
